doc/hardware/porting/board_porting.rst

```diff
@@ -67,27 +67,27 @@ complete conversion reference.
 
 .. _board_and_identifiers:
 
-Board and board identifiers
-***************************
+Board names, identifiers and targets
```

Comment:
qualifiers?

Reply:
Sorry, this is not ready for review yet. Working on it now